corehq/apps/hqwebapp/management/commands/show_invalid_bootstrap3_files.py

```diff
@@ -0,0 +1,180 @@
+from django.core.management import BaseCommand
+
+from corehq.apps.hqwebapp.utils.bootstrap.changes import (
+    flag_bootstrap3_references_in_template,
+    flag_bootstrap3_references_in_javascript, get_spec,
+)
+from corehq.apps.hqwebapp.utils.bootstrap.paths import (
+    get_all_template_paths_for_app,
+    get_all_javascript_paths_for_app,
+    get_short_path,
+)
+from corehq.apps.hqwebapp.utils.bootstrap.status import (
+    get_apps_completed_or_in_progress,
+    get_completed_templates_for_app,
+    get_completed_javascript_for_app,
+)
+from corehq.apps.hqwebapp.utils.management_commands import get_break_line
+
+
+IGNORED_FILES = [
```

Comment:
Why are these files ignored? I'm largely wondering if this is a list we're going to end up adding to over time.

Reply:
because files like `hqwebapp/base.html` includes references to both bootstrap 3 and bootstrap 5 files, acting as the starting point of the split

for instance:
https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/hqwebapp/templates/hqwebapp/base.html#L229-L233

I have a comment above the files I found were flagged by the test that I will update in a followup PR, as I didn't want to clutter this PR with those changes

Reply:
This list should not grow beyond the list before the comment I added. I can see web apps / formplayer templates potentially needing to included in this list.

Reply:
Wait, why would web apps templates need to be added?

Reply:
happy to go to voice if this is still unclear :)

Reply:
I see, so it would be base-level templates if anything, that's fine. I wouldn't be keen on adding all ~100 cloudcare templates.

Reply:
yes, exactly. this is a very small list for the starting points of the split files :)

Reply:
it only looks long right now because the test (accurately) discovered files that were missed by previous iterations of the migration tool which failed to catch files that had `include` or `static` references to `bootstrap3` files :)
The crispy forms list is a unique case, since we don't split the templates at the python level, but instead at the templates linked to each layout object